core/templates/dev/head/header_css_libs.html

```diff
@@ -15,4 +15,7 @@
 {% endif %}
 <link rel="stylesheet" type="text/css" media="screen"
       href="/css/oppia.css">
+<link rel="stylesheet" type="text/css" media="screen"
```

Comment:
You know -- this is quite a nice idea, and I don't know why I didn't think of it before. /cc @BenHenning

However, I think we should perhaps do this as a separate refactoring commit, and pull out all the overrides. For now, do you think we could just use oppia.css? And then, let's file an issue to refactor the third-party CSS out into a separate file.

Also, it's probably high time we get our CSS split up into context-specific files. Is this a refactoring project you would be interested in? If so, please file an issue for it, and let's discuss it there.

Thanks!